app/models/saved_claim/education_benefits/va_1995.rb

```diff
@@ -2,4 +2,46 @@
 
 class SavedClaim::EducationBenefits::VA1995 < SavedClaim::EducationBenefits
   add_form_and_validation('22-1995')
+
+  # Pulled from https://github.com/department-of-veterans-affairs/vets-website/src/applications/edu-benefits/utils/helpers.jsx#L100
+  # & fixtures at https://github.com/department-of-veterans-affairs/vets-website/src/applications/edu-benefits/1995/tests/e2e/
+  BENEFIT_TITLE_FOR_1995 = {
+    'chapter30' => 'Montgomery GI Bill (MGIB or Chapter 30) Education Assistance Program',
+    'chapter33Post911' => 'Post-9/11 GI Bill (Chapter 33)',
+    'chapter33FryScholarship' => 'The Fry Scholarship (Chapter 33)',
+    'chapter1606' => 'Montgomery GI Bill Selected Reserve (MGIB-SR or Chapter 1606) Educational Assistance Program',
+    'chapter32' => 'Post-Vietnam Era Veterans’ Educational Assistance Program (VEAP or chapter 32)',
```

Comment:
`Post-Vietnam Era Veterans’ Educational Assistance Program (VEAP or chapter 32)`

Reply:
Copy/paste error on my part. Should be `Post-Vietnam Era Veterans’ Educational Assistance Program (VEAP, chapter 32)`.